libraries/extractor/src/main/java/androidx/media3/extractor/mp3/Mp3InfoReplayGain.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package androidx.media3.extractor.mp3;
+
+import androidx.media3.common.Metadata;
+import androidx.media3.common.util.UnstableApi;
+import java.util.Objects;
+
+/** Representation of the ReplayGain data stored in a LAME Xing or Info frame. */
+@UnstableApi
+public final class Mp3InfoReplayGain implements Metadata.Entry {
+  /**
+   * 32 bit floating point "Peak signal amplitude".
+   *
+   * <p>1.0 is maximal signal amplitude store-able in decoding format. 0.8 is 80% of maximal signal
+   * amplitude store-able in decoding format. 1.5 is 150% of maximal signal amplitude store-able in
+   * decoding format.
+   *
+   * <p>A value above 1.0 can occur for example due to "true peak" measurement. A value of 0.0 means
+   * the peak signal amplitude is unknown.
+   */
+  public final float peak;
+
+  /**
+   * NAME of Gain adjustment in the first field, also called "Radio Replay Gain" field:
+   *
+   * <p>b000 = not set
+   *
+   * <p>b001 = radio
+   *
+   * <p>b010 = audiophile
+   */
+  public final byte field1Name;
```

Comment:
We don't need to perfectly match the spec here - it might be a clearer API to have an IntDef that reflects the three possible values of unset, radio & audiophile.

Similarly, maybe a nested type structure would be clearer than `name1A`, `name1B`, `name2A` etc.

Something like:

```java

public final class Mp3InfoReplayGain {

  public static final class GainField {
    public final @Name int name;
    public final @Originator int originator;
    public final int gain;
  }
  public final float peak;
  public final GainField field1;
  public final GainField field2;
}
```

Reply:
Thanks for the feedback, that makes a lot of sense and I implemented that.